sound/soc/sof/topology.c

```diff
@@ -2450,7 +2448,7 @@ static int sof_widget_ready(struct snd_soc_component *scomp, int index,
 			return -ENOMEM;
 		}
 
-		ret = sof_widget_load_dai(scomp, index, swidget, comp.core,
+		ret = sof_widget_load_dai(scomp, index, swidget,
```

Comment:
nitpick: some of these lines could be merged into 1

Reply:
> nitpick: some of these lines could be merged into 1

yes, I merged some of them but with some unchanged, I was hesitate about if I should follow the 80 characters or the new 100 rule when I was making the commit.

Let me follow the 100-character maximum rule and merge all of them if possible.